### PR TITLE
AT_004.01 | Entity > Placeholder > Input (New record with placeholder…

### PR DESCRIPTION
**Pull request by DanaDos**
AT_004.01.01 | Entity > Placeholder > Input (New record with placeholder values)
https://trello.com/c/cMxhduBB/402-at0040101-entity-placeholder-input-new-record-with-placeholder-values

### CODE REVIEW

**Review by omalanova — APPROVED** [review]

**Comment by OlenaKosenko** [comment]
AT Trello link needs to be in description.

File: src/test/java/EntityPlaceholder2Test.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import org.openqa.selenium.support.ui.Select;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import utils.TestUtils;
+
+import java.util.List;
+
+public class EntityPlaceholder2Test extends BaseTest {
+
+    private final String expectedString = "String placeholder";
```
Comment (OlenaKosenko) [inline src/test/java/EntityPlaceholder2Test.java:13]: 1. Wrong constant name style. Snake case needs to be used. Like EXPECTED_STRING
2. "static" is missing.

**Review by OlenaKosenko — CHANGES_REQUESTED** [review]
Constants needs to be updated. See comments in the code.

**Comment by DanaDos** [comment]
AT_004.01.01 | Entity > Placeholder > Input (New record with placeholder values) 
https://trello.com/c/cMxhduBB/402-at0040101-entity-placeholder-input-new-record-with-placeholder-values

File: src/test/java/EntityPlaceholder2Test.java (hunk 1 above)
Comment (DanaDos) [reply src/test/java/EntityPlaceholder2Test.java]: Thank you. Mistakes have been fixed.

**Review by omalanova — APPROVED** [review]
Good!

**Review by OlenaKosenko — DISMISSED** [review]

**Review by alla008 — APPROVED** [review]
LGFM

**Pull request merged**